en/docs/develop/service-provider-configurations-used-with-apis.md

```diff
@@ -139,19 +144,13 @@ This section guides you through the configurations you can include in a service
                 <li><b>idPInitSSOEnabled</b> <code>[Boolean]</code> <br>This is set <code>true</code> to enable IdP initiated SSO profile for the service provider.</li>
                 <li><b>idPInitSLOEnabled</b> <code>[Boolean]</code> <br>This is set <code>true</code> to enable IdP initiated SLO profile for the service provider.</li>
                 <li><b>assertionQueryRequestProfileEnabled</b> <code>[Boolean]</code> <br>This is set <code>true</code> to enable Assertion Query Request Profile to query assertions that are persisted to the database when you log in to the service provider application. For more information, see <a href="../../learn/querying-saml-assertions">Querying SAML Assertions</a>.</li>           
+                <li><b>idpEntityIDAlias</strong></b><code>[String]</code><br/>This value can override the value of Identity Provider Entity ID specified under SAML SSO Inbound Authentication configuration in Resident IdP. The Identity Provider Entity ID is used as the issuer of the SAML responses generated from IS. By default, all the SAML responses issued by IS will have the issuer value similar to the Identity Provider Entity ID in Resident IdP’s SAML SSO inbound authentication configuration. But if you want that value to be unique for your SAML SP configuration, you can specify the value here, so that the IdP Entity ID will be overridden with this IdP Entity ID Alias value.</li>
```

Comment:
This value can override `identity provider entity Id` that is specified under SAML SSO inbound authentication configuration of the resident identity provider. The Identity Provider Entity Id is used as the issuer of the SAML response that is generated by WSO2 Identity Server. By default, all the SAML responses that are issued by WSO2 Identity Server contain an issuer value that is similar to the `identity provider entity Id` of the resident identity provider's SAML SSO inbound authentication configuration. To make this value unique within the SAML service provider configuration, specify the value here causing the `identity provider entity Id` to be overridden by the `identity provider Id alias`.

Reply:
Fixed